runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -11287,18 +11293,21 @@ inlineConcurrentLinkedQueueTMOffer(
    static char * useNonConstrainedTM = feGetEnv("TR_UseNonConstrainedTM");
    static char * disableNIAI = feGetEnv("TR_DisableNIAI");
 
+   // the Transaction Diagnostic Block (TDB) is a memory location for the OS to write state info in the event of an abort
+   TR::MemoryReference* TDBmemRef = generateS390MemoryReference(cg->getMethodMetaDataRealRegister(), fej9->thisThreadGetTDBOffset(), cg);
+
    if (!disableTMOffer)
       {
       if (useNonConstrainedTM)
          {
-         /// immediate field described in TR::TreeEvaluator::tstartEvaluator(TR_Node * node, TR_CodeGenerator * cg)
-         cursor = generateSILInstruction(cg, TR::InstOpCode::TBEGIN, node, generateS390MemoryReference(cg->machine()->getS390RealRegister(TR::RealRegister::GPR0),0,cg), 0xFF02);
+         // immediate field described in TR::TreeEvaluator::tstartEvaluator
+         cursor = generateSILInstruction(cg, TR::InstOpCode::TBEGIN, node, TDBmemRef, 0xFF02);
 
          cursor = generateS390BranchInstruction(cg, TR::InstOpCode::BRC, TR::InstOpCode::COND_MASK7, node, failLabel);
          }
       else
-         {
-         cursor = generateSILInstruction(cg, TR::InstOpCode::TBEGINC, node, generateS390MemoryReference(cg->machine()->getS390RealRegister(TR::RealRegister::GPR0),0,cg), 0xFF00);  //GPRMask
+         { // No TDB for constrained transactions
```

Comment:
Comment is on the same line as the brace where as other comments are not. Can we be consistent with the rest of the codebase?

Reply:
This change is still missing. Can you squash after this and the one below is done and I'll kick off the Jenkins builds?

Reply:
It's here https://github.com/eclipse/openj9/pull/2548/commits/dfab63d5f5c8b8642886de00cd80a61e8d840b1f#diff-e91b45c91f84f8f7ea1179eb4ac01d36R11310

Reply:
Oh right, the 11533 line